### PR TITLE
#326-move-calendar-and-confluence-links

### LINKED ISSUE #326: Home Page - Move Confluence links to Homepage
### Description

apparently it's bad to link to the calendar in more than one place so we need to take it out of the info section and put the link to the confluence calendar page on the homepage

### Acceptance Criteria

* calendar links page block on info page is gone
* put a link on the home page for the calendars: https://nerdocs.atlassian.net/wiki/spaces/NER/pages/6619279/Calendars
* put a link on the home page for confluence: https://nerdocs.atlassian.net/wiki/spaces/NER/overview
* make sure they have good icons and stuff

### PR DESCRIPTION
**Pull request by NeilSatsangi**
## Changes

Removed calendar links block from info page
Added link on home page for calendars
Added link on home page for confluence
Inserted icons for both calendars and confluence
Changed all links in Useful Links block to open in a new tab

## Notes

I have verified that all links take you to the intended page

## Screenshots

Home Page
Info Page
Screen Shot 2023-02-22 at 11 02 16 AM
Screen Shot 2023-02-22 at 11 00 34 AM

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #326

### CODE REVIEW

**Review by ryanhowe28 — APPROVED** [review]
Looks good, looks like you merged develop into your branch you accepted changes that already removed the calender stuff from the info page?

**Review by RChandler234 — CHANGES_REQUESTED** [review]
I think you need to remove the calendar page block from the info page as well

File: src/frontend/src/pages/HomePage/UsefulLinks.tsx [hunk 1]
```diff
@@ -69,12 +82,37 @@ const UsefulLinks: React.FC = () => {
       <Settings sx={{ fontSize: 17, color: theme.palette.text.primary }} />
       <Link
         href="https://nerdocs.atlassian.net/wiki/spaces/NER/pages/4554841/Hardware+Guidelines"
+        target="blank"
         underline="hover"
         fontSize={19}
         sx={{ pl: 1 }}
       >
         Hardware Guidelines
       </Link>
```
Comment (ryanhowe28) [inline src/frontend/src/pages/HomePage/UsefulLinks.tsx:91]: Sorry i didnt pull when I looked at the new thing when you merged you added it back in

Comment (NeilSatsangi) [reply src/frontend/src/pages/HomePage/UsefulLinks.tsx:91]: I think it should be fixed now

**Review by ryanhowe28 — COMMENTED** [review]
Got tests failing

**Comment by NeilSatsangi** [comment]
> Got tests failing

Should be all set now!

**Pull request merged**